hstream/src/HStream/Server/Config.hs

```diff
@@ -479,3 +480,8 @@ metaStoreP = do
 instance Show MetaStoreAddr where
   show (ZkAddr addr) = "zk://" <> CB.unpack addr
   show (RqAddr addr) = "rq://" <> T.unpack addr
+
+readWithErrLog :: Read a => String -> String -> a
+readWithErrLog opt v = case readEither v of
+  Right x -> x
+  Left e -> errorWithoutStackTrace $ "Failed to parse value " <> show v <> " for option " <> opt
```

Comment:
The error value `e` seems unused here, would you like to mark it explicitly or display it in the error message?

Reply:
The err msg is not at all useful